app/views/layouts/application.html.erb

```diff
@@ -84,8 +84,7 @@
       <div class="usa-nav-container">
         <div class="usa-navbar">
           <div class="usa-logo" id="basic-logo">
-            <a href="/" accesskey="1" title="Home" aria-label="Home">
-              <%= image_tag 'img/login-gov-logo.svg', alt: 'login.gov', class: 'usa-logo__img' %>
+            <a href="/" title="Login.gov partner dashboard" aria-label="Login.gov partner dashboard">
               <em class="usa-logo__text"><%= Rails.application.config.app_name %></em>
```

Comment:
What is `Rails.application.config.app_name` -- is it different than the new `APP_NAME` constant?